src/main/java/byteceps/commands/ExerciseCommand.java

```diff
@@ -65,4 +67,39 @@ public CommandResult listExercises() {
             return new CommandResult(exerciseList.toString());
         }
     }
+
+    public CommandResult editExercise() {
+        String prevExerciseName = getActionParameters().trim();
+        ArrayList<InputArguments> newExercise = getAdditionalArguments();
+
+
+        if (prevExerciseName == null || prevExerciseName.isEmpty()) {
+            return new CommandResult("Previous exercise name cannot be empty.");
+        }
+
+        if (newExercise.size()!=1 ) {
```

Comment:
I think the formatting is not standard?
```suggestion
        if (newExercise.size() != 1) {
```

Reply:
Done